src/com/spikes2212/genericsubsystems/drivetrains/commands/DriveHolonomic.java

```diff
@@ -7,19 +7,46 @@
 import edu.wpi.first.wpilibj.command.Command;
 
 /**
- *
+ * This command moves a {@link HolonomicDrivetrain} using X and Y axes speeds.
  */
 public class DriveHolonomic extends Command {
 	protected final HolonomicDrivetrain holonomicDrivetrain;
 	protected final Supplier<Double> speedYSupplier, speedXSupplier;
 
+	/**
+	 * This constructs a new {@link DriveHolonomic} command that moves the given
+	 * {@link HolonomicDrivetrain} acording to constant X and Y axes speeds.<br>
+	 * Positive values move the axis forwards.
+	 *
```

Comment:
"positive values move **the drivetrain** forwards **on the corresponding axis**"